src/main/java/org/interledger/codecs/oer/ilp/ConditionOerCodec.java

```diff
@@ -0,0 +1,39 @@
+package org.interledger.codecs.oer.ilp;
+
+import org.interledger.Condition;
+import org.interledger.InterledgerAddress;
+import org.interledger.codecs.Codec;
+import org.interledger.codecs.CodecContext;
+import org.interledger.codecs.ConditionCodec;
+import org.interledger.codecs.oer.OerUint256Codec.OerUint256;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Objects;
+
+/**
+ * An implementation of {@link Codec} that reads and writes instances of {@link InterledgerAddress}.
```

Comment:
`InterledgerAddress` should be renamed to `Condition` in the javadoc. So, should read:

```
An implementation of {@link Codec} that reads and writes instances of {@link Condition}.
```